### PR TITLE
Create Simple_Edge_Discovery_API_Readiness_Checklist.md

### PR DESCRIPTION
**Pull request by Kevsy**

### CODE REVIEW

**Review by gunjald — COMMENTED** [review]
A query. Are there pre-defined values for Status field that needs to be referred uniformly for consistency purpose across API checklist?

**Comment by Kevsy** [comment]
> A query. Are there pre-defined values for Status field that needs to be referred uniformly for consistency purpose across API checklist?

No, only the table [here](https://github.com/camaraproject/Commonalities/blob/main/documentation/API-Readiness-Checklist.md)

File: documentation/API_documentation/Simple_Edge_Discovery_API_Readiness_Checklist.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Simple Edge Discovery (EDS) API Readiness minimum criteria checklist
+
+<br>
+
+| No | Deliverables/Criteria            | Mandatory | Status (Contributed/Approved/Pending/Validated/Partly-Validated)|
+|----|----------------------------------|-----------|----------------------------                         
+|  1 |API Spec                          |   Y       | Contributed                |
+|  2 |API Implementation                |   Y       | Contributed                |
```
Comment (javierlozallu) [inline documentation/API_documentation/Simple_Edge_Discovery_API_Readiness_Checklist.md:8]: As shown in [Commonalities](https://github.com/camaraproject/Commonalities/blob/main/documentation/API-Readiness-Checklist.md) API implementation is no longer mandatory.

File: documentation/API_documentation/Simple_Edge_Discovery_API_Readiness_Checklist.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+|  3 |API Documentation                 |   Y       | Contributed                |
+|  4 |User Stories                      |   Y       | Contributed                |
+|  5 |API test cases                    |   Y       | Contributed Gherkin feature |
+|  6 |Validated by atleast 2 operators  |   Y       | Partly validated (tested in two distinct VodaFone markets  |
```
Comment (javierlozallu) [inline documentation/API_documentation/Simple_Edge_Discovery_API_Readiness_Checklist.md:12]: @Kevsy, is the status still partially validated?

**Comment by Kevsy** [comment]
@javierlozallu I think we can say 'validated' because Vodafone have tested and so have Telstra at MWC (v0.9.3).

**Comment by gunjald** [comment]
> > A query. Are there pre-defined values for Status field that needs to be referred uniformly for consistency purpose across API checklist?
> 
> No, only the table [here](https://github.com/camaraproject/Commonalities/blob/main/documentation/API-Readiness-Checklist.md)

Thanks. I think it is fine for now.

**Review by gunjald — APPROVED** [review]
Looks fine from my side for merging.

**Pull request merged**